### DISCUSSION TITLE
Time Triggers - invoke URL

### DISCUSSION THREAD

**Discussion by larivierec** [fission/fission#3009] [Q&A] [answered]
Hello, I'm trying to convert one of my cluster Microservices to a serverless app. I've created a Time Trigger, an HTTPTrigger and a function based on a container. 

The container hosts multiple routes and I know that with the HTTP Trigger I'm able to hit those different routes using gateway-apis (I will share the example project when complete :)) however, now I have a TimeTrigger to continuously update a state instead of manually being invoked by an external source. 

Is it possible to invoke anything other than the root of your function with these Time Triggers?

**Comment by sanketsudake** [comment, marked as answer]
@soharab-ic made this possible with PR https://github.com/fission/fission/pull/3017

This would be available in upcoming release.

**Reply by larivierec** [reply]
excellent, excited to see this.
